app/ui/setup.py

```diff
@@ -284,11 +283,15 @@ def generate_code_to_create_security_integration(
 
 def generate_code_to_register_tenant(db: str, security_integration_name: str) -> str:
     return f"""
-let SecurityIntegration:='{security_integration_name}';
-let oauth_info variant := (parse_json(SYSTEM$SHOW_OAUTH_CLIENT_SECRETS(:SecurityIntegration)));
+let oauth_info variant := (parse_json(SYSTEM$SHOW_OAUTH_CLIENT_SECRETS('{security_integration_name}')));
 let tenantInfo object := {db}.tools.registertenant(:oauth_info:OAUTH_CLIENT_ID, :oauth_info:OAUTH_CLIENT_SECRET);
 CALL {db}.admin.setup_sundeck_tenant_url(:tenantInfo:sundeckTenantUrl, :tenantInfo:sundeckUdfToken);
-return OBJECT_CONSTRUCT('Sundeck Account ', tenantInfo:sundeckTenantUrl);
+
+let rs resultset := (select 'Go to Sundeck UI' As msg,
+    :tenantInfo:sundeckTenantUrl::string as url,
+    :tenantInfo:sundeckUdfToken::string as token);
```

Comment:
can we not show the token ? it's an internal detail that the customer doesn't need to be aware of

Reply:
done